Jellyfin.Server.Implementations/Item/PeopleRepository.cs

```diff
@@ -68,34 +69,41 @@ public IReadOnlyList<string> GetPeopleNames(InternalPeopleQuery filter)
     public void UpdatePeople(Guid itemId, IReadOnlyList<PersonInfo> people)
     {
         using var context = _dbProvider.CreateDbContext();
-        using var transaction = context.Database.BeginTransaction();
 
-        context.PeopleBaseItemMap.Where(e => e.ItemId == itemId).ExecuteDelete();
         // TODO: yes for __SOME__ reason there can be duplicates.
-        foreach (var item in people.DistinctBy(e => e.Id))
+        people = people.DistinctBy(e => e.Id).ToArray();
+        var personids = people.Select(f => f.Id);
+        var existingPersons = context.Peoples.Where(p => personids.Contains(p.Id)).Select(f => f.Id).ToArray();
```

Comment:
Have you considered using https://github.com/borisdj/EFCore.BulkExtensions for this?

Reply:
We looked at it but there are quite a lot of reports on silently failing operations and officially efcore 8 is the current supported version.

Reply:
I doubt that would make much of a difference. Overall we store like 8-10 persons per call

Reply:
It's not so much a speed optimization, but just utilizing the nice built-in features of the underlying SQL provider. SQLite supports UPSERT. Doing all these checks and manually removing this and that is just added complexity for an already-solved problem imo

Reply:
i am overall on board with the idea of introducing the BulkExtensions into JF, but i am not comfortable with doing it at this stage.